torch/csrc/jit/codegen/cuda/parser.cpp

```diff
@@ -1560,7 +1560,57 @@ class IrParser {
             nullptr);
       }
     }
+    {
+      if (isOptionEnabled(EnableOption::GraphOp)) {
+        auto ptr_op = getOperatorForLiteral(
+            "aten::gather(Tensor self, int dim, Tensor index, *, bool sparse_grad=False) -> Tensor");
+        REGISTER_PARSE_RULE(
+            ptr_op,
+            {
+              MemoryFormat format;
+              std::list<Val*> list_val;
+              std::tie(format, list_val) = getPWFormatValues(
+                  c10::nullopt,
+                  value_map[node->inputs()[0]->unique()],
+                  value_map[node->inputs()[2]->unique()]);
+              auto input = list_val.front();
+              list_val.pop_front();
+              auto dim_value = constant_as<int>(node->input(1));
+              TORCH_INTERNAL_ASSERT(
+                  dim_value.has_value(), "dim parameter is required.");
+              auto sparse_grad = constant_as<bool>(node->input(3));
+              TORCH_INTERNAL_ASSERT(
+                  sparse_grad.value() == false,
```

Comment:
I thought `sparse_grad` only affects backward ops?! Do we need to special case it here?!